docs/requirements.md

```diff
@@ -0,0 +1,9 @@
+---
+Order: 10
+Title: Requirements
+Description: Requirements for the Cake.Issues.EsLint addin.
+---
+The requirements for using the [Cake.Issues.EsLint addin] are listed in the [release notes] for any specific version.
+
+[Cake.Issues.EsLint addin]: https://www.nuget.org/packages/Cake.Issues.EsLint
+[release notes]: release-notes
```

Comment:
[MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.